source/includes/documentation/_vue_js_elements.md

```diff
@@ -2964,3 +2964,128 @@ Available methods in Java JDI Light:
 
 For examples of usage see: [JDI Vuetify Chip tests](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/ChipsTests.java).
 
+### 5.43 Inputs
+
+[Vuetify documentation page](https://vuetifyjs.com/en/components/images/)
+
+Input is located in the following class:
+- __Java__: _package com.epam.jdi.light.vuetify.elements.common.Input.java_
+
+```java
+    @Test(description = "Test checks type text feature")
+    public void typeTextInputTest() {
+        String textToType = "Some text";
+        hideDetailsAnotherInput.show();
+        hideDetailsAnotherInput.hasTextField();
+        hideDetailsAnotherInput.typeText(textToType);
+        hideDetailsAnotherInput.has().typedText();
+        hideDetailsAnotherInput.has().typedText(textToType);
+    }
+```
+
+__Input__ - The v-input component gives you a baseline to create your own custom inputs. It consists of a prepend/append slot, messages, and a default slot.
+
+![Input example](../../images/vuetify/input.png)
+
+__Vuetify v2.6.14__ code example:
+```html
+<template>
+  <v-container
+    id="input-usage"
+    fluid
+  >
+    <v-row>
+      <v-col cols="12">
+        <v-input
+          :messages="['Messages']"
+          append-icon="mdi-close"
+          prepend-icon="mdi-phone"
+        >
+          Default Slot
+        </v-input>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script>
+  export default {
+    data () {
+      return {
+        text: '',
+      }
+    },
+    methods: {
+      appendIconCallback () {},
+      prependIconCallback () {},
+    },
+  }
+</script>
+
+<style>
+  #input-usage .v-input__prepend-outer,
+  #input-usage .v-input__append-outer,
+  #input-usage .v-input__slot,
+  #input-usage .v-messages {
+    border: 1px dashed rgba(0,0,0, .4);
+  }
+</style>
+```
```

Comment:
Please, change according to Vladimir's comments:
in a documentation near highlighted comment please also emphasize a Selenium locator findby 
![image](https://user-images.githubusercontent.com/55189259/212973313-914b69e7-0912-4abe-af57-c62784090e4e.png)